robot_skills/src/robot_skills/sergio.py

```diff
@@ -4,8 +4,15 @@
 
 
 class Sergio(robot.Robot):
-    """docstring for Sergio"""
+    """
+    Sergio
+    """
     def __init__(self, wait_services=False):
+        """
+        Constructor
+
+        :param wait_services: Not supported anymore by robot class
```

Comment:
Can we issue a deprecation warning when this is used?

Reply:
We can provide a deprecation warning if the value is `True`. But it needs a lot of refactoring to determine if it set by the user or not in case of `False`.

Reply:
OK, it's a start. There's only a few cases where we use it anyways I expect